R/tags.R

```diff
@@ -211,7 +211,15 @@ as.character.html <- function(x, ...) {
 }
 
 #' @export
-print.shiny.tag.list <- print.shiny.tag
+print.shiny.tag.list <- function(x, ...) {
+  if (isTRUE(attr(x, "print.as.list", exact = TRUE))) {
+    attr(x, "print.as.list") <- NULL
+    class(x) <- setdiff(class(x), "shiny.tag.list")
+    return(print(x))
```

Comment:
Does it work if you just call `return(NextMethod())` instead of these lines?

Reply:
It does work, but it has ugly print output...

```r
tagQuery(div())$selectedTags()
#> [[1]]
#> <div></div>
#> 
#> attr(,"class")
#> [1] "shiny.tag.list" "list"
#> attr(,"print.as.list")
#> [1] TRUE
```

-------------------

Going to keep it as is